eng/scripts/Language-Settings.ps1

```diff
@@ -89,16 +89,33 @@ function Get-dotnet-PackageInfoFromPackageFile ($pkg, $workingDirectory)
   }
 }
 
+# Return list of nupkg artifacts
+function Get-dotnet-Package-Artifacts ($Location)
+{
+  $pkgs = Get-ChildItem "${Location}" -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+  if (!$pkgs)
+  {
+    Write-Host "$($Location) does not have any package"
+    exit(1)
```

Comment:
Similar comment to java PR we shouldn't exit from a helper function we should simply return "$null" and bubble the error up the stack to the older script.

Reply:
Fixed